pywbem/cim_operations.py

```diff
@@ -486,6 +487,32 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     the :attr:`last_raw_request` and :attr:`last_raw_reply` attributes
     of the connection object.
 
+    WBEMConnections can record the results of methods that interact with
```

Comment:
On "WBEMConnections": Don't use a class name in plural. Either use "WBEM connections" or "WBEMConnection objects"

Reply:
Changed to WBEMConnection objects